frontend/src/components/App/PluginSettings/PluginSettings.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import store from '../../../redux/stores/store';
+import { TestContext } from '../../../test';
+import { PluginSettingsPure, PluginSettingsPureProps } from './PluginSettings';
+
+export default {
+  title: 'Settings/PluginSettings',
+  component: PluginSettingsPure,
+} as Meta;
+
+const Template: Story<PluginSettingsPureProps> = args => (
+  <TestContext store={store}>
+    <PluginSettingsPure {...args} />
+  </TestContext>
+);
+
+/** createDemoData function will create example data objects to act as plugin data
```

Comment:
This needs updating to follow the doc convention. There should be a newline added before createDemoData. See other documented functions. Other documentation in this PR also needs updating.

Reply:
From https://tsdoc.org/
![image](https://user-images.githubusercontent.com/9541/228823195-8f5e3702-7fbc-4b3d-9b50-d2267a360813.png)

See how the first line is written?